client/tsconfig.json

```diff
@@ -12,9 +12,13 @@
     "moduleResolution": "node",
     "resolveJsonModule": true,
     "isolatedModules": true,
-    "noEmit": true,
+    "incremental": true,
+    "declaration": true,
+    "emitDeclarationOnly": true,
+    "outDir": "dist",
     "types": ["node", "@types/jest", "@testing-library/jest-dom"],
     "jsx": "preserve"
   },
-  "include": ["src"]
+  "include": ["src"],
+  "exclude": ["**/node_modules", "**/.*/"]
```

Comment:
Yes, we were type checking all of node_modules before 🤦

Reply:
🙈

Reply:
What, really? I'd expect TypeScript to know not to do this.

Reply:
Well, I'm not 100% sure actually. I think I remember adding that exclude line and rerunning the lint-staged command and seeing a big speedup. I got the tip from this doc: https://github.com/microsoft/TypeScript/wiki/Performance#misconfigured-include-and-exclude.

But reading closer in this other section of the doc, it says node_modules is excluded by default: https://github.com/microsoft/TypeScript/wiki/Performance#specifying-files. So maybe I was wrong or not diligent in only changing one setting at a time.